docs/_toc.yml

```diff
@@ -18,6 +18,7 @@
     - file: content-types/restructuredtext
 
 - header: Write book content
+- file: content/cheatsheet
```

Comment:
Can we nest this underneath the myst page? https://deploy-preview-637--jupyter-book.netlify.app/content/myst.html

or put it in the "Reference" section? I feel like this is a good and complete reference-style page

Reply:
resolved here 30f1e23